pjsip/src/pjsua-lib/pjsua_media.c

```diff
@@ -3001,6 +3001,7 @@ pj_status_t pjsua_media_channel_deinit(pjsua_call_id call_id)
 	    call_med->tp = call_med->tp_orig = NULL;
 	}
         call_med->tp_orig = NULL;
+        call_med->rem_srtp_use = PJ_FALSE;
```

Comment:
should be set to `PJMEDIA_SRTP_DISABLED`? or create new enum `UNKNOWN` (==`DISABLED`) & set to this.

Reply:
ok